### ISSUE TITLE
Add to some package repository

### ISSUE THREAD

**Issue by noctuid** [abo-abo/short-lambda#1]
Would be nice to have on MELPA or in some repository.

**Comment by abo-abo** [comment]
Were there any improvements to Emacs that allow the user to customize the reader macros? If not, I'm not sure `short-lambda` deserves its name:

```elisp
(mapcar (short-lambda (* %1 %1)) (number-sequence 0 9))
;; vs:
(mapcar (lambda (x) (* x x)) (number-sequence 0 9))
```

My old proposal to modify the C core to add just the `#()` syntax got rejected.

**Comment by noctuid** [comment]
No, I don't think so. It looks like tarsius also created a similiar library just called [l](https://git.sr.ht/~tarsius/l).
